src/presentation/react/core/components/sync-wizard/common/SummaryStep.tsx

```diff
@@ -216,32 +228,49 @@ export const SummaryStepContent = (props: SummaryStepContentProps) => {
                 </ul>
             </LiEntry>
 
-            {_.keys(metadata).map(metadataType => {
-                //@ts-ignore
-                const modelByMetadataType = api.models[metadataType];
-                if (!modelByMetadataType) {
-                    console.warn(`Metadata type "${metadataType}" not supported in d2-api`);
-                    return null;
-                }
-                const itemsByType = metadata[metadataType as keyof MetadataEntities] || [];
+            {_(metadata)
+                .keys()
+                .concat(syncRule.metadataSyncAll)
+                .uniq()
+                .sort()
+                .value()
+                .map(metadataType => {
+                    //@ts-ignore
```

Comment:
It's better to a casting `as keyof MetadataEntities` than ts-ignore. Of course, the better would be not to have neither :)